avalon/tools/loader/model.py

```diff
@@ -17,15 +17,18 @@ def is_filtering_recursible():
 
 
 class SubsetsModel(TreeModel):
-    Columns = ["subset",
-               "family",
-               "version",
-               "time",
-               "author",
-               "frames",
-               "duration",
-               "handles",
-               "step"]
+    Columns = [
+        "subset",
+        "asset",
```

Comment:
This seems new too. Any reason for having this in this particular PR?

Reply:
No, this is from future PR...